src/assets/scss/_reset.scss

```diff
@@ -0,0 +1,97 @@
+* {
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+.img-responsive {
+  width: 100%;
+}
+.list-none {

```

Comment:
make mixin for this class because we have multiple reset values in this one and we always have to reuse this ul reset so its an important mixin.